apps/sports-helsinki/src/domain/search/combinedSearch/CombinedSearch.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  EventTypeId,
+  useLocale,
+  useSearchTranslation,
+} from 'events-helsinki-components';
+import { useRouter } from 'next/router';
+import qs from 'query-string';
+import React from 'react';
+import { ContentContainer, PageSection } from 'react-helsinki-headless-cms';
+import { ROUTES } from '../../../constants';
+import { getI18nPath } from '../../../utils/routerUtils';
+import type { AdvancedSearchProps } from '../eventSearch/AdvancedSearch';
+import AdvancedSearch from '../eventSearch/AdvancedSearch';
+import styles from './combinedSearch.module.scss';
+import { SearchTab } from './searchUtilities/SearchTabs';
+import SearchUtilities from './searchUtilities/SearchUtilities';
+
+export type SearchResultItemsCount = {
+  [EventTypeId.Course]: number | null;
+  [EventTypeId.General]: number | null;
+  Venue: number | null;
+};
+
+export type CombinedSearchProps = {
+  searchStatus: SearchResultItemsCount;
+} & AdvancedSearchProps;
+
+const CombinedSearch: React.FC<CombinedSearchProps> = ({
+  searchStatus,
+  ...delegatedProps
+}) => {
+  const { t } = useSearchTranslation();
+  const router = useRouter();
+  const locale = useLocale();
+  const params: { eventType?: string } = router.query;
+  const activeSearchEventType = params.eventType
+    ? (params.eventType as EventTypeId)
+    : null;
+
+  const switchToGeneralEventSearch = () => {
```

Comment:
pretty much the same implementation as method below, could it be some generic used with eventtypeid param?